app/src/main/java/com/infomaniak/mail/ui/main/thread/actions/ActionsBottomSheetDialog.kt

```diff
@@ -1,6 +1,6 @@
 /*
  * Infomaniak kMail - Android
- * Copyright (C) 2022-2023 Infomaniak Network SA
+ * Copyright (C) 2023 Infomaniak Network SA
```

Comment:
Do we want this ?

Reply:
Yes it's technically a new file, the old one has been renamed